easybuild/easyconfigs/i/Inspector/Inspector-2017_update1.eb

```diff
@@ -13,4 +13,5 @@ dontcreateinstalldir = 'True'
 
 license_file = HOME + '/licenses/intel/license.lic'
 
+modextrapaths = {'PATH' : ['inspector/bin64'] }
```

Comment:
@shahzebsiddiqui we need a similar fix for 2016 update3, but with `inspector_xe/bin64`, so we should handle this in the `EB_Inspector` easyblock instead?